api/src/main/java/marquez/MarquezConfig.java

```diff
@@ -38,4 +39,8 @@
   @Getter
   @JsonProperty("flyway")
   private final FlywayFactory flywayFactory = new FlywayFactory();
+
+  public static class GraphqlConfig {
```

Comment:
Moving the graphql configuration to the `graphql` package will help group all graphql related code.